AzureCalling/External/Calling/CallingContext.swift

```diff
@@ -129,6 +131,11 @@ class CallingContext: NSObject {
                 self.callType = joinConfig.callType
                 self.displayName = joinConfig.displayName
                 self.isCameraPreferredOn = joinConfig.isCameraOn
+
+                self.callRecordingFeature = self.call?.api(RecordingFeature.self)
+                self.callTranscriptionFeature = self.call?.api(TranscriptionFeature.self)
```

Comment:
Do you have the link to the change log?

Reply:
https://github.com/Azure/Communication/releases/